pkg/loadbalancer/experimental/bpf_reconciler.go

```diff
@@ -601,6 +610,31 @@ func (ops *BPFOps) updateFrontend(fe *Frontend) error {
 		}
 	}
 
+	// Update Maglev
+	if ops.cfg.NodePortAlg == option.NodePortAlgMaglev {
+		ops.log.Info("Update Maglev", "feID", feID)
```

Comment:
Should be `Debug`, but let's do an explicit separate pass to change all the `Info`s to `Debug`s.